app/AppKernel.php

```diff
@@ -27,8 +28,11 @@ public function registerBundles()
             new Sylius\Bundle\GridBundle\SyliusGridBundle(),
             new Sylius\Bundle\UiBundle\SyliusUiBundle(),
 
+            new Sonata\CoreBundle\SonataCoreBundle(),
+            new Sonata\BlockBundle\SonataBlockBundle(),
```

Comment:
this bundles optional? Or required sylius register to kernel?

Reply:
Required, most of syliusUI templates has sonata blocks